core/services/workflows/cmd/cre/examples/v2/simple_cron_with_config/main.go

```diff
@@ -0,0 +1,47 @@
+//go:build wasip1
+
+package main
+
+import (
+	"fmt"
+
+	croncap "github.com/smartcontractkit/chainlink-common/pkg/capabilities/v2/triggers/cron"
+	"github.com/smartcontractkit/chainlink-common/pkg/workflows/sdk/v2"
+	"github.com/smartcontractkit/chainlink-common/pkg/workflows/wasm/v2"
+	"gopkg.in/yaml.v2"
+)
+
+type runtimeConfig struct {
+	Schedule string `yaml:"schedule"`
+}
+
+func RunSimpleCronWorkflow(runner sdk.DonRunner) {
+	cron := &croncap.Cron{}
+	cfg := &croncap.Config{
+		Schedule: "*/3 * * * * *", // every 3 seconds
```

Comment:
This example with CRON schedule in the config doesn't make sense if we're not reading it here.
Should we have access to workflow config in the Runner so we can use it here? (I think so).
Otherwise maybe provide a different example with a config that is useful at runtime. Ideally you could showcase both.